builder/builder.go

```diff
@@ -515,10 +515,10 @@ func (b *Builder) saveArtifactLocally(ctx context.Context, artifact domain.Artif
 				}
 			} else {
 				// Remove pre-existing dest dir.
-				err = os.RemoveAll(to)
-				if err != nil {
-					return errors.Wrapf(err, "rm -rf %s", to)
-				}
+				// err = os.RemoveAll(to)
+				// if err != nil {
+				// 	return errors.Wrapf(err, "rm -rf %s", to)
+				// }
```

Comment:
This doesn't look right.

The core idea is that if the artifact is a directory it should come out locally as that same directory, and not as a merged version of it+whatever already exists. Reason: If a build is being repeated and the new output no longer includes a certain file, then that file should now be removed.

Reply:
I wasnt quite ready for a review yet :D Yall are quick right now?

I question the need for cleanup; but maybe this means that a (`--no-cleanup`/`--merge`) option would be nice. For sure there have been times I wanted that; say comingling earthly-made builds with local made builds.

Reply:
ALso this was here just to see what the CI would think since I still havent set up locally to run tests with the new QEmu dependency